pkg/kinflate/util/labelsandannotations.go

```diff
@@ -22,35 +22,41 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
-type ApplyAdditionalMapOptions struct {
-	additionalMap map[string]string
-	pathConfigs   []PathConfig
+// MapTransformationOptions contains a map string->string and path configs
+// The map will be applied to the fields specified in path configs.
+type MapTransformationOptions struct {
```

Comment:
Please call this MapTransformer, or TransformerByMap, or something.  

Its an implementation of Transformer.